django/contrib/gis/gdal/driver.py

```diff
@@ -49,7 +49,7 @@ def __init__(self, dr_input):
 
             # Attempting to get the GDAL/OGR driver by the string name.
             for iface in (vcapi, rcapi):
-                driver = iface.get_driver_by_name(force_bytes(name))
+                driver = c_void_p(iface.get_driver_by_name(force_bytes(name)))
```

Comment:
This change is needed because GDALBase doesn't accept `int` as value for `ptr` property anymore.